static/src/js/util/accessMethods/__tests__/buildAccessMethods.test.js

```diff
@@ -686,14 +933,54 @@ describe('buildAccessMethods', () => {
       }
       const isOpenSearch = false
 
-      const methods = buildAccessMethods(collectionMetadata, isOpenSearch)
+      buildAccessMethods(collectionMetadata, isOpenSearch)
 
-      expect(methods).toEqual({})
+      expect(buildSwodlrMock).toBeCalledTimes(1)
+
+      expect(buildSwodlrMock).toHaveBeenCalledWith({
+        conceptId: 'S100000-EDSC',
+        longName: 'Mock PODAAC SWOT On-Demand Level 2 Raster Generation (SWODLR)',
+        name: 'Mock PODAAC_SWODLR',
+        orderOptions: {
+          items: []
+        },
+        serviceOptions: {
+          supportedOutputProjections: [
+            {
+              projectionName: 'Universal Transverse Mercator'
+            },
+            {
+              projectionName: 'WGS84 - World Geodetic System 1984'
+            }
+          ]
+        },
+        supportedInputProjections: null,
+        supportedOutputProjections: [
+          {
+            projectionName: 'Universal Transverse Mercator'
+          },
+          {
+            projectionName: 'WGS84 - World Geodetic System 1984'
+          }
+        ],
+        supportedReformattings: null,
+        type: 'SWODLR',
+        url: {
+          description: 'Service top-level URL',
+          urlValue: 'https://swodlr.podaac.earthdatacloud.nasa.gov'
+        },
+        variables: {
+          items: []
+        }
+      })
     })
   })
 
   describe('when the collection contains both variables associated to its services and variables directly associated to the collection', () => {
-    test('variables on the service are returned instead of variables directly associated to the collection', () => {
+    test('variables on the service are returned instead of variables directly associated to the collection and buildHarmony is called 3 times', () => {
```

Comment:
`and buildHarmony is called 3 times` is confusing, that doesn't happen because of what the describe block is saying, it is just because there are 3 service records in the test. Describe blocks should describe the input and the test block should explain what happens because of that input